src/components/Footer/Footer.js

```diff
@@ -1,7 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+import FooterDataModal from '../Modal/FooterDataModal';
+import {
+  StyleFooter,
+  FooterContainer,
+  FooterInner,
+  FooterWrapper,
+  Ul,
+  Li,
+  FooterDiv,
+  SlideDiv,
+  MapBtn,
+  MapBtnSpan,
+  MapBtnLink,
+  FooterWrapperDiv,
+  FooterLink,
+} from './StyleFooter';
```

Comment:
```suggestion
import * as S from './StyleFooter';
```
이런식으로 수정해주세요!
자세한 내용은 필터 UI에 있는 내용을 참고해주세요